node/node.go

```diff
@@ -108,12 +107,13 @@ type Node struct {
 	ConfirmedBlockChannel chan *types.Block    // The channel to send confirmed blocks
 	BeaconBlockChannel    chan *types.Block    // The channel to send beacon blocks for non-beaconchain nodes
 	DRand                 *drand.DRand         // The instance for distributed randomness protocol
-	pendingCrossLinks     []*block.Header
-	pendingClMutex        sync.Mutex
 
 	pendingCXReceipts map[string]*types.CXReceiptsProof // All the receipts received but not yet processed for Consensus
 	pendingCXMutex    sync.Mutex
 
+	pendingCrossLinks []types.CrossLink
```

Comment:
Let's not use in-memory cache anymore. Our normal txns are already using TxPool, which I believe keeps txns in local db for persistence.

Reply:
will add the tmp flag back to local db persistency